frontend/www/js/omegaup/arena/submissions.ts

```diff
@@ -5,6 +5,10 @@ import * as time from '../time';
 import { types } from '../api_types';
 import { myRunsStore, runsStore } from './runsStore';
 import { OmegaUp } from '../omegaup';
+import JSZip from 'jszip';
+import problem_Details from '../components/problem/Details.vue';
```

Comment:
```suggestion
import type problem_Details from '../components/problem/Details.vue';
```

esto nada más se está usando en declaraciones, así que no es necesario importarlo "completo".

Reply:
Listo.